### ISSUE TITLE
No longer functioning with patch 5.2

### ISSUE THREAD

**Issue by HealsCodes** [NotAdam/ffxiv-unstrip-acls#1]
I only now found this fix and I'd love to use it to get my discord overlay back without requiring admin privileges but it seems 5.2 (or possibly earlier, couldn't test) broke this overload.

When placing `winmm.dll` into the `boot/` directory and staring `ffxivboot.exe` everything works as planned, the launcher opens and I can login, however once I click play the launcher closes and instead of `ffxiv_dx11` I get an error popup stating 

> "Unable to launch application. Launch using ffxivboot.exe file(0)".

I tried both with the release binary from github and with a version I compiled from the sources.

**Comment by NotAdam** [comment]
Yeah SE actually added a targeted fix just to stop this from working and I just haven't got around to fixing it yet, don't really play this game anymore.

I believe https://github.com/goaaats/FFXIVQuickLauncher does fix this in the interim if you'd like to use that cause I'm not totally sure when I'll get around to fixing this

**Comment by NotAdam** [comment]
It's actually a pretty simple fix, I might suss it out later today if I don't get distracted doing something else
